evm/src/executor/inspector/cheatcodes/ext.rs

```diff
@@ -70,12 +70,11 @@ fn get_code(path: &str) -> Result<Bytes, Bytes> {
 
     let mut buffer = String::new();
     fs::File::open(path)
```

Comment:
I just added `foundry_common::fs` which provides wrapper functions for `std::fs` that also return more info about in the error 

forgot to update `fs` usage in `evm` crate...

this can be replaced with `foundry_common::fs::read_to_string().map_err..`

Reply:
fixed